### ISSUE TITLE
Cannot use in Jiayu S3

### ISSUE THREAD

**Issue by DickyT** [ikarus23/MifareClassicTool#65]
Hi I own a Chinese brand phone Jiayu S3 NFC version. Tech detail here
http://forum.xda-developers.com/android/general/jiayu-s3-5-5-inch-1080p-hd-4g-lte-t3015809

and when i was reading my tag with 100% correct keys, i got 'no valid key found in chosen key file' and i cannot continuing reading my card. Is MCT not yet supported on MTK phones?
Looking for solution, thank you very much.

BTW my os is Android 5.1 with latest MCT

**Comment by moscowneversleeping** [comment]
@DickyT Do you try use MCT on your device with Android 4.*?

**Comment by ikarus23** [comment]
Hi, do you know what kind of NFC chip is inside this phone? Also, did you try other apps like [NFC Taginfo](https://play.google.com/store/apps/details?id=at.mroland.android.apps.nfctaginfo&hl=en), [NFC TagInfo by NXP](https://play.google.com/store/apps/details?id=com.nxp.taginfolite&hl=en) or [NFC Tools](https://play.google.com/store/apps/details?id=com.wakdev.wdnfc)

**Comment by DickyT** [comment]
Hi, I flashed back to 4.4 official version yesterday, and I found MCT also cannot use. And I tried NFC TagInfo by NXP, it can read my Mifare Classic 1K card as "MIFARE Classic (MF1S50)" with IC manufacturer is NXP Semiconductors, i think my phone have a fully support on Mifare Classic Cards. And I also tried NFC Tools Pro and NFC TagInfo both can read my card UID, but NFC TagInfo also cannot decrypt my card and show HEX using provided Keys.

Thank you~

**Comment by DickyT** [comment]
P.S. The NXP TagInfo also cannot read my Tag HEX data using given Keys.

**Comment by ikarus23** [comment]
Hi, I'm sorry but it seams like the phone does not support Mifare Classic.
I will add it to the list of unsupported devices.

**Comment by DickyT** [comment]
Hi there, I did some research about this issue, and I found may be this issue is caused by the MTK customized Android, because there is one App call ALIPAY, it can read the Authorized Bus Pass and shows the transaction history and the credit inside. I tried my cards, on of them is Sony Felica and other is Mifare 1K Classic.

**Comment by ikarus23** [comment]
This is weird. So you are able to read a Mifare Classic 1k tag? But only with the ALIPAY App?

**Comment by jekkos** [comment]
It seems that extra suppport for mifare classic might be on it's way (in AOSP)

https://android-review.googlesource.com/#/c/163641/

**Comment by DickyT** [comment]
Hi there, I found some important informations here. In June, my phone
reseller and the customer agent in that company said to me that Jiayu S3
uses NXP chips, but today I found while I entering the Factory Mode of this
phone, and I found the chip inside this MTK phone is MT6605, which is by
MTK. Although I did not found lots of informations of this chip, but i
think I can say that is chip do not fully support Mifare M1 cards.

2015-08-20 23:16 GMT+08:00 jekkos <notifications@github.com>:
